security/pkg/nodeagent/plugin/providers/google/stsclient/stsclient_test.go

```diff
@@ -75,6 +75,7 @@ func TestGetFederatedToken(t *testing.T) {
 		tag := map[string]string{
 			"request_type": monitoring.TokenExchange,
 		}
+		// TODO: this is broken due to prometheus auto adding "_total"
 		mt.Assert("num_outgoing_retries", tag, monitortest.AtLeast(1))
```

Comment:
is this worth a release notes?

Reply:
Stale comment  - its not broken anymore. Let me clean this up